packages/api/src/functions/secureHandler.test.ts

```diff
@@ -0,0 +1,61 @@
+import { ForbiddenError } from 'apollo-server-lambda'
```

Comment:
We might want to use a generic error error, since I would more than likely not use this in an GraphQL context.

Reply:
Done! Created WebhookError and WebhookVerificationError. Classes are inline.

Reply:
Added WebhookVerificationError